.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
   "cSpell.import": ["cspell.yaml"],
-  "go.testFlags": ["-timeout", "5m"]
+  "go.testFlags": ["-timeout", "60m"]
```

Comment:
60m seems a little long for local tests, but maybe this is okay

Reply:
Some of the functional tests take more than 5 min.. Let's split the different and go with 30m to align with above.